python/cudf/cudf/tests/test_series.py

```diff
@@ -1813,3 +1813,17 @@ def test_series_digitize_invalid_bins():
         ValueError, match="`bins` cannot contain null entries."
     ):
         _ = s.digitize(bins)
+
+
+@pytest.mark.parametrize("data", [[0, 1, 2, 3, 4, 5, 10]])
```

Comment:
Do we also need to test other dtypes? This works already for strings. 😄
```python
>>> s = cudf.Series(["a", "few", "strings"])
>>> (s >= "banana") & (s <= "phone")
0    False
1     True
2    False
dtype: bool
```

Reply:
Added coverage for this 😉